dotnet/src/webdriver/WebDriver.csproj

```diff
@@ -117,6 +117,10 @@
       <Visible>False</Visible>
       <LogicalName>webdriver_prefs.json</LogicalName>
     </EmbeddedResource>
+    <EmbeddedResource Include="$(ProjectDir)..\..\..\common\manager\windows\selenium-manager.exe">
```

Comment:
As I understand it includes exe file embedded into dll. Given that how many target frameworks you support, total size of nuget package will increase significantly:

![image](https://user-images.githubusercontent.com/22616990/198146743-70dda605-6f65-4c8e-8be6-a182481c13eb.png)

There is easier way. Include exe file anewhere in nuget package, and then copy this file to anywhere while user compiles his project (using msbuild tagets).

Reply:
Yeah; the real question is whether we can do that with Bazel...

Reply:
I think yes. Bazel uses `nuspec` definition file to create nuget package. Here we can include any files like image is already included `<file src="../icon.png" target="images\" />`. The same we can use to include `*.exe` and `*.targets` files.

Reply:
So, I built the nupkg and it went from 5MB to 15MB, so not storing it for each framework, thankfully.